Testscripts/Linux/SetupHbKernel.sh

```diff
@@ -20,18 +20,16 @@ GetDistro
 # Get generation: 1/2
 GetGuestGeneration
 
-# Hibernation is supported in RHEL-8 since kernel-4.18.0-202. Not supported in RHEL-7.
-if [[ $DISTRO =~ "redhat" ]];then
-	MIN_KERNEL="4.18.0-202"
-	CheckVMFeatureSupportStatus $MIN_KERNEL
-	if [[ $? == 1 ]];then
-		UpdateSummary "Hibernation is supported since kernel-4.18.0-202. Current version: $(uname -r). Skip the test."
-		SetTestStateSkipped
-		exit 0
+function Main() {
+	basedir=$(pwd)
+	if [[ "$DISTRO" =~ "redhat" || "$DISTRO" =~ "centos" ]];then
+		# RHEL requires bigger disk space for kernel repo and its compilation.
+		# This is Azure mnt disk from the host.
+		linux_path=/mnt/linux
```

Comment:
resource disk is different when use cloud-init and waagent

Reply:
Yes, but this is working with this specific VM size and distro. I don't know how it changes later.